src/gdpr-banner/gdpr-banner.scss

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../dialog/index';
+
+@mixin iui-gdpr-banner {
+  @include iui-dialog;
+  border-radius: 0;
+  box-shadow: none;
+  background-color: var(--iui-color-background-5);
```

Comment:
Let's make a slight visual change to increase the contrast score of the text against the background. 

```suggestion
  background-color: var(--iui-color-background-3);
  border-top: 1px solid var(--iui-color-background-border);
```

Because the background color is getting lighter with this change, I think we need to make the division from the body content and the GDPR banner more prominent.  Adding `border-top: 1px solid var(--iui-color-background-border);` should help with that.

**Results after this change:**
![Screen Shot 2022-05-25 at 1 49 06 PM](https://user-images.githubusercontent.com/849817/170329893-274bf059-341a-4499-b1a5-4ccf596feb66.png)
![Screen Shot 2022-05-25 at 1 49 36 PM](https://user-images.githubusercontent.com/849817/170329962-aaae8b4f-54d1-4034-8dad-4189e491803c.png)

Reply:
Made these adjustments